docs/data-modeling/metadata-editing.md

```diff
@@ -141,6 +141,15 @@ You can manually change the user interface for the filter to:
 
 ![Filter options](./images/filter-options.png)
 
+## Changing a search box filter to a dropdown filter
```

Comment:
I couldn't decide where to put this info --- it used to live in the Filters troubleshooting guide.

It's about _setting up_ a filter (so there's a link on `/dashboards/filters.md` above), but the actual changes need to be done on the Data Model page.

Does this make sense, or is there a more intuitive way to organize this?